### DISCUSSION TITLE
Need a bit of help troubleshooting https

### DISCUSSION THREAD

**Discussion by HenningNT** [datalust/seq-tickets#1643] [Help] [answered]
I have 3 servers running Seq, and have set up https successfully  on 2 of them. It's the last one I need help with.
The server team set up the servers, and created certificates for the servers. I've bound the certificates, and added the routing.
I've also tried setting up https on a different port.

All servers have listenUris like so:
"listenUris": [
  "https://servername.domain.com/seq",
  "http://localhost:5341"
],
(I've checked the spelling, it's OK)

When I browse to https://server.domain.com/seq I get an error message:

> Hmmm… can't reach this pageIt looks like servername.domain.com closed the connection
> Try:
> 
> Checking the connection
> [Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
> [Running Windows Network Diagnostics](javascript:diagnoseErrors())
> ERR_CONNECTION_CLOSED

If I try to open the url in a browser running on the server locally:

> This site can’t be reached 
> servername.domain.com refused to connect.
> Try:
> 
> Checking the connection
> [Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
> ERR_CONNECTION_REFUSED

I was thinking that the firewall or something was preventing the connection, but it still should have worked locally.

Any help would be greatly appreciated!

**Comment by nblumhardt** [comment, marked as answer]
Thanks for getting in touch, Henning. This still looks pretty consistent with what I'd expect from a firewall; I'm not 100% confident Windows Firewall can be configured to block local connections but I believe it's possible. Perhaps adding an exception for port 443 would be a good next step to try?

**Reply by HenningNT** [reply]
Thanks!, I'll ask the server admins if they can have a look at the firewall.

**Reply by HenningNT** [reply]
Looks like the firewall is not to blame.
I ran netstat -an, which lists all open and listening ports.
Nothing is listening to port 433. I tried to bind ssl to port 44333, and that did not show up either.

Would ssl bind failure show in the internal seq log?

**Reply by HenningNT** [reply]
Don't ask me how, but I have apparently managed to install Seq multiple times in more than one location. I was editing the wrong seq.json file...

**Reply by HenningNT** [reply]
It's working now.

**Reply by nblumhardt** [reply]
Great! Thanks for the follow-up, Henning :+1:
